src/components/permission/PermissionGroupAdminItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { EditOutlined } from '@ant-design/icons'
+import { Button } from 'antd'
+import { useIntl } from 'react-intl'
+import styled from 'styled-components'
+import { permissionGroupsAdminMessages } from '../../helpers/translation'
+import PermissionGroupAdminModal from './PermissionGroupAdminModal'
+import PermissionGroupDeletionModal from './PermissionGroupDeletionModal'
+
+const StyledAdminBlock = styled.div`
+  margin-bottom: 1.25rem;
+  padding: 1rem 1.5rem;
+  background: white;
+  border-radius: 4px;
+  box-shadow: 0 4px 12px 0 rgba(0, 0, 0, 0.06);
+`
+
+const PermissionGroupAdminItem: React.VFC<{
+  id: string
+  name: string
+  permissionGroupPermissions?: string[]
```

Comment:
why this prop is optional?
`permissionIds` might be better naming

Reply:
```ts
const permissionGroup = {
   members: [],
   orders: [],
   permissions: [],
   permissionIds: [],
}
permissionGroup.members -> // object[]
permissionGroup.permissions -> // object[]
permissionGroup.permissionIds -> // string[]
```

Reply:
```ts
const memberPhones = {
   memberId: string
   phone: string
}

const member = {
    phones: data.memberPhones.map(memberPhone => memberPhone.phone)
}

permission: id, group, description
currency: id, name
phone: string
member_phone: object

// maybe in the future
phone: country_code, number, created_at, ...
```